### PR TITLE
[TG-101][feat] Scenario Editor page: styling

### PR DESCRIPTION
**Pull request by oartyshko**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/23" target="_blank">ReviewNinja

File: src/app/scenarios/scenario-editor/scenario-editor.component.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    initCodeEditor() {
+        this.codeEditor = this.codeMirror(
+            this.codeEditorElement.nativeElement,
+            {
+                mode: 'javascript',
+                lineNumbers: true,
+                autoCloseBrackets: true,
+                matchBrackets: true,
+                tabSize: 2,
```
Comment (kosich) [inline src/app/scenarios/scenario-editor/scenario-editor.component.js]: 4, please? 😢 
https://github.com/garage-it/SmartHouse-frontend/edit/story/TG-101/src/app/scenarios/scenario-editor/scenario-editor.component.js?pr=/garage-it/SmartHouse-frontend/pull/23

File: src/app/app.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500,700);
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
```
Comment (kosich) [inline src/app/app.scss:2]: since our site is deployed privately -- we would gain more if we use local stored fonts

**Comment by kosich** [comment]
⭐

File: src/app/scenarios/scenario-editor/scenario-editor.component.js (hunk 1 above)
Comment (oartyshko) [inline src/app/scenarios/scenario-editor/scenario-editor.component.js]: ok. it was copy-paste (blushing)

File: src/app/app.scss (hunk 2 above)
Comment (oartyshko) [inline src/app/app.scss:2]: ok, I will change it later today

**Pull request merged**